en/docs/learn/tutorials/expose-a-soap-service-as-a-rest-api.md

```diff
@@ -42,7 +42,25 @@ This tutorial will explain the steps to design, publish and invoke a SOAP servic
 4.  The created API appears in the publisher as follows.
     ![]({{base_path}}/assets/img/learn/created-soap-api.jpg)
 
-5.  API definiton of the Created schema has been displayed at **API Definiton** tab.
+5.  Add below API definiton manually for Created schema by clicking on **Edit** button under **API Definiton** tab.
```

Comment:
5. Click **Edit** under the **API Definition** tab and replace the API definition with the below for the created schema.

Reply:
> 1. Click **Edit** under the **API Definition** tab and replace the API definition with the below in place of the created schema.

We just need to add the below content in API definition from line number 12 to 27 (refer attached doc). Replacing the whole API definition is not required.
![api-definition](https://user-images.githubusercontent.com/10825796/89877142-4af44800-dbdd-11ea-92b7-096a81d57a34.jpg)

Reply:
@akurathimohan 
This content seems to be already there in the generated swagger definition. Could you please double check?

Reply:
Yes, this content is already there in document. But, this swagger definition is not generating automatically (as mentioned in doc) when we create Phoneverify API on our machines, we have to add this definition manually by clicking on "Edit" button under "API Definiton" tab.